Description: Adding testing.sh script and postinst message
 .
 hello (2.10-2ubuntu5) jammy; urgency=medium
 .
   * Added testing.sh script and postinst message printing
     "this is a test from Jayeshkumar Natvarlal Patel"
Author: Jayesh Patel <jnpatel81@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2025-03-19

--- /dev/null
+++ hello-2.10/testing.sh
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "this is a test from Jayeshkumar Natvarlal Patel" >&2
